external/mbedtls/Kconfig.protocol

```diff
@@ -12,6 +12,12 @@ config SUPPORT_FULL_SECURITY
 	---help---
 		Supports security features with secure storage.
 
+config TLS_MPI_MAX_SIZE
+	int "TLS MPI Max Size"
+	default 512
+	---help---
+		Read certificate support MPI max size.
```

Comment:
Is this enough description to know how to determin the value of this config?

Reply:
Explanation supplemented.